src/components/GlobalLearningResourcesPage/GlobalLearningResourcesContent.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect } from 'react';
+import { TabContent } from '@patternfly/react-core';
+import './GlobalLearningResourcesContent.scss';
+import { Bullseye, Gallery } from '@patternfly/react-core';
+import { QuickStart } from '@patternfly/quickstarts';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import {
+  Button,
+  EmptyState,
+  EmptyStateActions,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+} from '@patternfly/react-core';
+import CubesIcon from '@patternfly/react-icons/dist/esm/icons/cubes-icon';
+import fetchSuperQuickstarts from '../../utils/fetchQuickstarts';
+import GlobalLearningResourcesQuickstartItem from './GlobalLearningResourcesQuickstartItem';
+import { useSearchParams } from 'react-router-dom';
+
+type Foobar<AsyncFunc extends (...args: any[]) => Promise<unknown>> = (
+  ...args: Parameters<AsyncFunc>
+) => Awaited<ReturnType<AsyncFunc>>;
+
+interface GlobalLearningResourcesContentProps {
+  purgeCache: () => void;
+  loader: Foobar<typeof fetchSuperQuickstarts>;
```

Comment:
I think this should be imported from the FEC utils suspense loader. `UnwrappedLoader`